pkg/blockchain/substrate.go

```diff
@@ -373,10 +373,18 @@ func (s *SubstrateService) TotalHotkeyStake(hotkey string) (float64, error) {
 	params := url.Values{}
 	params.Add("keys[]", hotkey)
 	storageResponse, err := s.GetStorageRequest(path, params)
-	if err != nil {
-		log.Error().Err(err).Msgf("Error getting total hotkey stake for hotkey %s", hotkey)
-		return 0, err
+
+	// If first attempt fails, try TotalHotkeyAlpha
+	if err != nil || storageResponse == nil || storageResponse.Value == nil {
+		log.Debug().Msgf("TotalHotkeyStake not found for %s, trying TotalHotkeyAlpha", hotkey)
+		path = fmt.Sprintf("%s/pallets/subtensorModule/storage/TotalHotkeyAlpha", s.substrateApiUrl)
```

Comment:
TotalHotkeyAlpha has extra argument for subnetUID, which you can get from outer function.
This is the example call `http://35.240.182.230:8080/pallets/subtensorModule/storage/TotalHotkeyAlpha?keys[]=5CBFxnF2MKYrS6LMr48wQTFhqmch2G9pS3AQkFm4RF8Jd1qe&keys[]=98`